### PR TITLE
Update hackrf.c

### PR DESCRIPTION
**Pull request by JEremy3535**
It's very easy to make some words **bold** and other words *italic* with Markdown. You can even [link to Google!](http://google.com

### CODE REVIEW

**Comment by JEremy3535** [comment]
host/libhackrf/src/hackrf.c

**Comment by JEremy3535** [comment]
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\bin\x86_amd64\Lib.exe /OUT:"C:\projects\hackrf\host\build\hackrf-tools\src\Release\libgetopt_static.lib" /NOLOGO /MACHINE:X64  /machine:x64 libgetopt_static.dir\Release\getopt.obj
  libgetopt_static.vcxproj -> C:\projects\hackrf\host\build\hackrf-tools\src\Release\libgetopt_static.lib
AppLocalFromInstalled:
  C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe -ExecutionPolicy Bypass -noprofile -File "C:\Tools\vcpkg\scripts\buildsystems\msbuild\applocal.ps1" "C:\projects\hackrf\host\build\hackrf-tools\src\Release\libgetopt_static.lib" "C:\Tools\vcpkg\installed\x64-windows\bin" "libgetopt_static.dir\Release\libgetopt_static.tlog\libgetopt_static.write.1u.tlog" "libgetopt_static.dir\Release\vcpkg.

**Comment by dmaltsiniotis** [comment]
This appears to be at best some kind of mistake pull request or at worst some kind of spam/deception to test if PRs are automatically/blindly accepted...

File: host/libhackrf/src/hackrf.c [hunk 1]
```diff
@@ -13,7 +13,7 @@ Redistribution and use in source and binary forms, with or without modification,
     Neither the name of Great Scott Gadgets nor the names of its contributors may be used to endorse or promote products derived from this software
 	without specific prior written permission.
```
Comment (JEremy3535) [inline host/libhackrf/src/hackrf.c:14]: N

**Pull request closed without merging**